.github/workflows/tests.yml

```diff
@@ -6,6 +6,8 @@ on:
       - mirnetv2/**
       - tests.py
       - requirements.txt
+env:
+  WANDB_API_KEY: ${{ secrets.WANDB_API_KEY }}
```

Comment:
Why do we need an API key for the tests ? All the tests have `visualize_on_wandb` set to `False` ?

Reply:
@SauravMaheshkar We need the API key for fetching the artifacts from wandb.

Reply:
Ahh yes ofcourse got it 👍🏽